android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,51 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Movie
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+@Composable
+fun Moviecard(
+    item: Movie,
+    modifier: Modifier = Modifier,
+    onMovieItemClick: () -> Unit = {}
+) {
+    Card(
+        modifier = modifier
+            .width(122.dp)
+            .height(179.dp)
```

Comment:
Width and Height should not be hardcoded. 
In general hardcoding both width and height is a bad idea since this makes the component fixed in size. In android the user can change the system text size in OS settings which would impact how much content fits into this component rendering some parts invisible or some text unreadable. 
Additionally, the component size should be handled by the caller of this function since the component is not aware of the context (in this case layout/screen) in which it will be used in.